qemu/tests/virtio_scsi_mq.py

```diff
@@ -172,15 +155,14 @@ def proc_interrupts_results(results):
         if not dev:
             continue
         if not re.findall("[svh]d%s" % system_dev, dev):
-            fill_cmd += " dd of=%s if=/dev/urandom bs=1M " % dev
-            fill_cmd += "count=%s &" % dd_timeout
+            fill_cmd += " dd of=%s if=/dev/urandom bs=4k count=%s oflag=direct & " % (dev, dd_timeout*256)
```

Comment:
why do you changed bs and count here?

Reply:
Just want to the system generate much more interrupters so that the numbers of interrupts is more obvious.